settings.php

```diff
@@ -811,6 +818,41 @@ function convertseconds($argument)
                                                    DNSSEC. Note that the size of your log might increase significantly
                                                    when enabling DNSSEC. A DNSSEC resolver test can be found
                                                    <a href="http://dnssec.vs.uni-due.de/" target="_blank">here</a>.</p>
+                                                <label>Conditional Forwarding</label>
+                                                <p>If not configured as your DHCP server Pi-Hole won't able to
+                                                   determine the names of devices on your local network and tables
```

Comment:
Some more commas would be helpful to the reader, such as between `server` and `Pi-hole`, and `network` and `and`.